version.go

```diff
@@ -38,6 +45,19 @@ func setupVersion(cfg *config.Config, osd *osd.OSD) (err error) {
 			cfg.MajorTarget = -1
 		}
 
+		if len(cfg.TargetStream) != 0 {
+			log.Printf("Looking for a release on stream %s", cfg.TargetStream)
+			if cfg.ClusterVersion, _, err = upgrade.LatestRelease(cfg, cfg.TargetStream); err == nil {
+				if !strings.Contains(cfg.ClusterVersion, "openshift-v") {
```

Comment:
It seems like we could make this into a function and combine the above logic with it.

Reply:
absolutely